src/app/reporting/reporting.cpp

```diff
@@ -694,53 +694,6 @@ EmberStatus emAfPluginReportingRemoveEntry(uint8_t index)
     return status;
 }
 
-void emberAfReportingAttributeChangeCallback(EndpointId endpoint, ClusterId clusterId, AttributeId attributeId, uint8_t mask,
```

Comment:
Please do not include old code cleanup until later changes after we get this landed and only focus on the core of this change to make the delta to enable subs easier to process.

When we go to share, we should also try to upload this change on top of a merged PR from Yunhan on a branch and do code change as one commit, followed by test changes as another commit to ease looking at code delta.

Reply:
This is the only (and necessary) removal to get code compiled, or we will have two `emberAfReportingAttributeChangeCallback` and linker will be unhappy.

Reply:
Yes, but why do you have to replicate the ember name? I gather something is calling that function an that's why you are just replacing it so the linker picks up your version, but couldn't you name your function differently and replace the name at the call site of emberAfReportingAttributeChangeCallback? Is that call site not available to you?